### ISSUE TITLE
opens opera:debug page. No response in a timely fashion

### ISSUE THREAD

**Issue by Jenka156** [operasoftware/operaprestodriver#59]
Hello! I hope someone can help me with my problem.
I use selenium 2.17.0
Run my tests on virtual mashine with windows 7
Opera version 11.60

When I run test in opera, it opens opera:debug page and nothing more. After some time it falls with org.openqa.selenium.WebDriverException: No response in a timely fashion.

I check opera:config. There is enable Proxy Auto Connect.

How can I solve this problem?

**Comment by andreastt** [comment]
Have you set any special capabilities, such as `opera.profile` or `opera.port`?  If you modify those two, it will in some cases not be able to figure out which Opera to connect to automatically.

Does launching Opera take over 30 seconds on your system?  There's a 30 second time out before it throws that exception.

**Comment by Jenka156** [comment]
no, I haven't set any capabilities.

And opera runs fast on my system. I turned off all opera widgets

**Comment by andreastt** [comment]
Do you have OperaDriver installed through Maven?  Can you set the capability `opera.logging.level` to `FINE` and paste the output, please?

```
DesiredCapabilities c = DesiredCapabilities.opera();
c.setCapability("opera.logging.level", "FINE");
WebDriver driver = new OperaDriver(c);
```

**Comment by Jenka156** [comment]
Driver info: driver.version: OperaDriver
16:25:33.445 INFO - Executing: [new session: {platform=ANY, browserName=opera, o
pera.logging.level=FINE, version=}] at URL: /session)
16:25:33.445 INFO - Executing: [new session: {platform=ANY, browserName=opera, o
pera.logging.level=FINE, version=}] at URL: /session)
16:25:33.460 CONFIG - Opera arguments: [-pd, C:\Users\kvm\AppData\Local\Temp\ope
ra441064478486841998profile, -debugproxy, 127.0.0.1:14909, -autotestmode]
16:25:33.460 CONFIG - Opera arguments: [-pd, C:\Users\kvm\AppData\Local\Temp\ope
ra441064478486841998profile, -debugproxy, 127.0.0.1:14909, -autotestmode]
16:25:33.476 CONFIG - launcher arguments: [-host, 127.0.0.1, -port, 16257, -cons
ole, -verbosity, INFO, -bin, C:\Program Files (x86)\Opera\opera.exe, -pd, C:\Use
rs\kvm\AppData\Local\Temp\opera441064478486841998profile, -debugproxy, 127.0.0.1
:14909, -autotestmode]
16:25:33.476 CONFIG - launcher arguments: [-host, 127.0.0.1, -port, 16257, -cons
ole, -verbosity, INFO, -bin, C:\Program Files (x86)\Opera\opera.exe, -pd, C:\Use
rs\kvm\AppData\Local\Temp\opera441064478486841998profile, -debugproxy, 127.0.0.1
:14909, -autotestmode]
16:25:33.523 DEBUG - Waiting for launcher connection on port 16257
16:25:33.523 DEBUG - Waiting for launcher connection on port 16257
16:25:34.599 DEBUG - Got launcher connection from /127.0.0.1:49336
16:25:34.599 DEBUG - Got launcher connection from /127.0.0.1:49336
16:25:34.615 DEBUG - Connected with launcher on port 16257
16:25:34.615 DEBUG - Connected with launcher on port 16257
16:25:34.615 CONFIG - Capabilities [{platform=ANY, opera.binary=C:\Program Files
 (x86)\Opera\opera.exe, javascriptEnabled=true, opera.host=127.0.0.1, opera.idle
=false, browserName=opera, opera.autostart=true, opera.logging.file=null, opera.
guess_binary_path=true, opera.logging.level=FINE, opera.profile=com.opera.core.s
ystems.OperaProfile@5b247d84, version=, opera.launcher=null, opera.display=null,
 opera.product=core-gogi, opera.arguments=, opera.no_quit=false, opera.port=1490
9, opera.no_restart=false}]
16:25:34.615 CONFIG - Capabilities [{platform=ANY, opera.binary=C:\Program Files
 (x86)\Opera\opera.exe, javascriptEnabled=true, opera.host=127.0.0.1, opera.idle
=false, browserName=opera, opera.autostart=true, opera.logging.file=null, opera.
guess_binary_path=true, opera.logging.level=FINE, opera.profile=com.opera.core.s
ystems.OperaProfile@5b247d84, version=, opera.launcher=null, opera.display=null,
 opera.product=core-gogi, opera.arguments=, opera.no_quit=false, opera.port=1490
9, opera.no_restart=false}]
16:25:34.630 DEBUG - Instructing launcher to start Opera...
16:25:34.630 DEBUG - Instructing launcher to start Opera...
16:25:34.630 DEBUG - opera: Missing -id argument, settig id to: 127.0.0.1:16257
16:25:34.630 DEBUG - opera: Missing -id argument, settig id to: 127.0.0.1:16257
16:25:34.630 DEBUG - launcher: trying to start opera in thread...
16:25:34.630 DEBUG - launcher: trying to start opera in thread...
16:25:37.146 DEBUG - opera: WinLauncher::starting opera
16:25:37.146 DEBUG - opera: WinLauncher::starting opera
16:25:37.146 DEBUG - opera: WinLauncher::path: C:\Program Files (x86)\Opera\oper
a.exe
16:25:37.146 DEBUG - opera: WinLauncher::path: C:\Program Files (x86)\Opera\oper
a.exe
16:25:37.148 DEBUG - opera: WinLauncher::arg:  -pd C:\Users\kvm\AppData\Local\Te
mp\opera441064478486841998profile -debugproxy 127.0.0.1:14909 -autotestmode
16:25:37.148 DEBUG - opera: WinLauncher::arg:  -pd C:\Users\kvm\AppData\Local\Te
mp\opera441064478486841998profile -debugproxy 127.0.0.1:14909 -autotestmode
16:25:37.151 DEBUG - opera: WinLauncher::process started...
16:25:37.151 DEBUG - opera: WinLauncher::process started...
16:25:37.156 DEBUG - launcher: successfull starting opera!
16:25:37.156 DEBUG - launcher: successfull starting opera!
16:25:37.246 DEBUG - Opera launched through launcher
16:25:37.246 DEBUG - Opera launched through launcher
16:26:07.247 DEBUG - Shutting down STP connection listener
16:26:07.247 DEBUG - Shutting down STP connection listener
16:26:15.935 DEBUG - OperaDriver shutting down
16:26:15.935 DEBUG - OperaDriver shutting down
16:26:16.184 DEBUG - Shutting down launcher
16:26:16.184 DEBUG - Shutting down launcher
16:26:17.229 WARN - Exception thrown
java.util.concurrent.ExecutionException: org.openqa.selenium.WebDriverException:
 java.lang.reflect.InvocationTargetException
Build info: version: '2.17.0', revision: '15540', time: '2012-01-16 16:51:46'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.ver
sion: '1.7.0_01'
Driver info: driver.version: unknown
        at java.util.concurrent.FutureTask$Sync.innerGet(Unknown Source)
        at java.util.concurrent.FutureTask.get(Unknown Source)
        at org.openqa.selenium.remote.server.DefaultSession.execute(DefaultSessi
on.java:139)
        at org.openqa.selenium.remote.server.DefaultSession.<init>(DefaultSessio
n.java:108)
        at org.openqa.selenium.remote.server.DefaultSession.createSession(Defaul
tSession.java:86)
        at org.openqa.selenium.remote.server.DefaultDriverSessions.newSession(De
faultDriverSessions.java:87)
        at org.openqa.selenium.remote.server.handler.NewSession.handle(NewSessio
n.java:62)
        at org.openqa.selenium.remote.server.rest.ResultConfig.handle(ResultConf
ig.java:196)
        at org.openqa.selenium.remote.server.DriverServlet.handleRequest(DriverS
ervlet.java:577)
        at org.openqa.selenium.remote.server.DriverServlet.doPost(DriverServlet.
java:528)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
        at org.openqa.selenium.remote.server.DriverServlet.service(DriverServlet
.java:500)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
        at org.openqa.jetty.jetty.servlet.ServletHolder.handle(ServletHolder.jav
a:428)
        at org.openqa.jetty.jetty.servlet.ServletHandler.dispatch(ServletHandler
.java:677)
        at org.openqa.jetty.jetty.servlet.ServletHandler.handle(ServletHandler.j
ava:568)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1530)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1482)
        at org.openqa.jetty.http.HttpServer.service(HttpServer.java:909)
        at org.openqa.jetty.http.HttpConnection.service(HttpConnection.java:820)

```
    at org.openqa.jetty.http.HttpConnection.handleNext(HttpConnection.java:9
```

86)
        at org.openqa.jetty.http.HttpConnection.handle(HttpConnection.java:837)
        at org.openqa.jetty.http.SocketListener.handleConnection(SocketListener.
java:243)
        at org.openqa.jetty.util.ThreadedServer.handle(ThreadedServer.java:357)
        at org.openqa.jetty.util.ThreadPool$PoolThread.run(ThreadPool.java:534)
Caused by: org.openqa.selenium.WebDriverException: java.lang.reflect.InvocationT
argetException
Build info: version: '2.17.0', revision: '15540', time: '2012-01-16 16:51:46'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.ver
sion: '1.7.0_01'
Driver info: driver.version: unknown
        at org.openqa.selenium.remote.server.DefaultDriverFactory.callConstructo
r(DefaultDriverFactory.java:73)
        at org.openqa.selenium.remote.server.DefaultDriverFactory.newInstance(De
faultDriverFactory.java:57)
        at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(
DefaultSession.java:177)
        at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(
DefaultSession.java:1)
        at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)

```
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)

    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Sou
```

rce)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.server.DefaultDriverFactory.callConstructo
r(DefaultDriverFactory.java:63)
        ... 8 more
Caused by: org.openqa.selenium.WebDriverException: com.opera.core.systems.scope.
exceptions.ResponseNotReceivedException: No response in a timely fashion
Build info: version: '2.17.0', revision: '15540', time: '2012-01-16 16:51:46'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.ver
sion: '1.7.0_01'
Driver info: driver.version: OperaDriver
Build info: version: '2.17.0', revision: '15540', time: '2012-01-16 16:51:46'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.ver
sion: '1.7.0_01'
Driver info: driver.version: OperaDriver
        at com.opera.core.systems.OperaDriver.start(OperaDriver.java:358)
        at com.opera.core.systems.OperaDriver.<init>(OperaDriver.java:312)
        ... 13 more
Caused by: com.opera.core.systems.scope.exceptions.ResponseNotReceivedException:
 No response in a timely fashion
Build info: version: '2.17.0', revision: '15540', time: '2012-01-16 16:51:46'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.ver
sion: '1.7.0_01'
Driver info: driver.version: OperaDriver
        at com.opera.core.systems.WaitState.pollResultItem(WaitState.java:435)
        at com.opera.core.systems.WaitState.waitAndParseResult(WaitState.java:46
4)
        at com.opera.core.systems.WaitState.waitForHandshake(WaitState.java:665)

```
    at com.opera.core.systems.ScopeServices.waitForHandshake(ScopeServices.j
```

ava:323)
        at com.opera.core.systems.ScopeServices.init(ScopeServices.java:210)
        at com.opera.core.systems.OperaDriver.init(OperaDriver.java:377)
        at com.opera.core.systems.OperaDriver.start(OperaDriver.java:354)
        ... 14 more
16:26:17.416 WARN - Exception: No response in a timely fashion
Build info: version: '2.17.0', revision: '15540', time: '2012-01-16 16:51:46'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.ver
sion: '1.7.0_01'
Driver info: driver.version: OperaDriver
16:26:17.447 DEBUG - Handled by org.openqa.jetty.jetty.servlet.ServletHandler@33
c063ff
16:26:17.447 DEBUG - Handled by org.openqa.jetty.jetty.servlet.ServletHandler@33
c063ff
16:26:17.463 DEBUG - RESPONSE:
HTTP/1.1 500 Internal Server Error
Date: Tue, 24 Jan 2012 13:25:33 GMT
Server: Jetty/5.1.x (Windows 7/6.1 amd64 java/1.7.0_01
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Cache-Control: no-cache
Content-Length: 5874
Content-Type: application/json;charset=UTF-8

16:26:17.463 DEBUG - RESPONSE:
HTTP/1.1 500 Internal Server Error
Date: Tue, 24 Jan 2012 13:25:33 GMT
Server: Jetty/5.1.x (Windows 7/6.1 amd64 java/1.7.0_01
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Cache-Control: no-cache
Content-Length: 5874
Content-Type: application/json;charset=UTF-8

16:26:17.697 DEBUG - EOFException()
16:26:17.697 DEBUG - EOFException()

**Comment by Jenka156** [comment]
Today I got this exeption: 
org.openqa.selenium.WebDriverException:
Error 500 java.util.concurrent.ExecutionException: java.lang.StackOverflowError
HTTP ERROR: 500 java.util.concurrent.ExecutionException: java.lang.StackOverflowError
RequestURI=/wd/hub/session/1327473421551
Powered by Jetty://

After it i run  test with selenium 2.16.1 and it opens URL. But when my test fall I got this exeption:

FAILED CONFIGURATION: @AfterMethod stopWebDriver
org.openqa.selenium.WebDriverException: Error on command: service: "window-manager"
commandID: 1
format: 0
status: 4
tag: 129
payload: "\n\020No active window"

**Comment by Jenka156** [comment]
I solved this problem. just build new selenium server with OperaDriver10
all works

**Comment by srividya-su** [comment]
I am facing the same issue as Jenka is this issue fixed with 11.6 can any one help me out.

**Comment by srividya-su** [comment]
I am using opera driver 0.14
